modules/elasticsearch/elasticsearch_test.go

```diff
@@ -108,21 +106,19 @@ func TestElasticsearch(t *testing.T) {
 
 			if tt.image != baseImage8 && err != nil {
```

Comment:
We probably need to restructure this if/else sequence here, as it was not already very readable before the changes, and the readability does not improve after them.